src/UIAutomation/FunctionalTests/UiaOperationAbstractionTests.cpp

```diff
@@ -11,6 +11,57 @@
 
 using namespace UiaOperationAbstraction;
 
+// Test framework extensions used for comparing and printing different types via the
+// `Assert` test functions.
+namespace Microsoft::VisualStudio::CppUnitTestFramework
+{
+#pragma warning(push)
+#pragma warning(disable: 4505)
+    template<>
+    std::wstring ToString(const std::vector<int>& vec)
+#pragma warning(pop)
+    {
+        std::wstringstream ss;
+        ss << L"[";
+
+        for (size_t i = 0; i < vec.size(); ++i)
+        {
+            if (i != 0)
+            {
+                ss << L", ";
+            }
+
+            ss << vec[i];
+        }
+
+        ss << L"]";
+        return ss.str();
+    }
+
+#pragma warning(push)
+#pragma warning(disable: 4505)
+    template<>
+    std::wstring ToString(const std::map<std::wstring, int>& map)
```

Comment:
>oString(const std::map<std::wstring, int>& map) [](start = 18, length = 47)

Future thinking: This could be more generic since the types start to multiply.